h5c/vic/src/vic-webapp/src/app/create-vch-wizard/compute-capacity/compute-capacity.component.ts

```diff
@@ -1,3 +1,5 @@
+import 'rxjs/add/observable/timer';
```

Comment:
Is this actually intended to go about the copyright header?

Reply:
Ah, it's my VSCode extension that did it. I just turned it off completely, as it has been more of annoyance than help. Will address this too.